### DISCUSSION TITLE
Config: Increase default resolution limit from 100 to 150 MP

### DISCUSSION THREAD

**Discussion by TheIceMagmaCube** [photoprism/photoprism#2677] [Q&A]
Hi, 
I'm using Photoprism on OpenMediaVault and it works fine, but Photoprism doesn't load my highres pictures. The files are quite heavy (more than 20Mo/pic) and for an unknown reason the system doesn't show them in the explorer. It show others pictures, but not these HQ pictures (12000*9000).

Tbh I don't have any idea why this doesn't work. It can import files and class them, but don't show them in the interface.

Thanks for your help!

**Comment by lastzero** [comment]
The resolution is limited to 100 MP by default via `PHOTOPRISM_RESOLUTION_LIMIT`, see https://docs.photoprism.app/getting-started/config-options/#storage. The reason for this is that we have frequently received support requests from users who ran out of memory after indexing very large images.

**Comment by TheIceMagmaCube** [comment]
Hi, can we change this limit easily on a config using openmediavault?
Thanks for your help!

**Reply by lastzero** [reply]
We could increase the value by default if it turns out that 100 MP is a common size - this specific limit was just an assumption based on the maximum resolution of cameras. As I said, the main goal is to avoid crashes, which then lead to many support requests or discussions. These keep us from developing new features, which then leads to unhappy users.

When it comes to openmediavault specifically, it would be best to ask their community how it works, because we are not the maintainers and I'm not sure how many openmediavault users are on our GitHub discussion forum.

**Reply by lastzero** [reply]
Would a 150 MP limit work for you and most others? That can easily be changed, you just have to wait until the next release or use our preview.

**Reply by lastzero** [reply]
Done, see https://github.com/photoprism/photoprism/commit/810914a578b8f5ed4c01206a8406ee945b2ac653.

**Comment by lastzero** [comment]
We have added the resolution limits for the personal editions in the [feature overview on our website](https://photoprism.app/editions#compare):

![Screenshot 2022-09-18 at 14 27 14](https://user-images.githubusercontent.com/301686/190902110-1d987fe0-9412-4aaa-a835-931f5ac69850.png)

- the 150 MP limit is set to avoid support intensive crashes of our completely free version and to protect the hosted service from denial of service attacks
- the 900 MP limit for our sponsors is a technical limit, as our JPEG implementation does not support more than 30,000 x 30,000 pixels

*Note that you need to wait for the upcoming release, as in the current stable release the default limit is 100 MP. Sponsors can increase it manually by changing the configuration.*

**Comment by TheIceMagmaCube** [comment]
I installed PhotoPrism in OpenMediaVault, there is a way to change it on this kind of installations?

4 Sep 2022 11:52:36 Michael Mayer ***@***.***>:

**Reply by graciousgrey** [reply]
As far as we know, only a few [config options](https://docs.photoprism.app/getting-started/config-options/) are available in the openmediavault UI, I am not sure if the resolution limit is one of them. 

If it is available in the OMV UI and you have sponsor features enabled you should be able to increase it. 
Otherwise you would need to wait until we've had the chance to make some more config options configurable using PhotoPrism's UI.
